assembly/runtime/contract.ts

```diff
@@ -57,8 +57,7 @@ class Context {
   }
 
   /**
-  * The amount of tokens received with this execution call.
-  * @deprecated use attachedDeposit.
+  * Returns the current balance of the sender account.
   */
```

Comment:
i think it was `accountBalance` that has been deprecated and `attachedDeposit` that should be commented

Reply:
Any conclusion here?

Reply:
I would assume @janedegtiareva would reply since it's her edit. I was just reading the code and comments, seems like we mixed up the methods. am I wrong? if so just disregard

Reply:
just looked again.  

my read is that `accountBalance` is the deprecated method so removing the `@deprecated` notice from line `61` is a mistake. 

![image](https://user-images.githubusercontent.com/24913/79902525-4d5a4880-841a-11ea-9507-7f9fdc1e352d.png)

Reply:
ah, ok, just looked again and it seems like this is my mistake.  the `@deprecated` warning was copy/pasted to an unrelated method.  please ignore